test/asmt-double-entry.js

```diff
@@ -34,8 +74,8 @@ describe('navigate to asmt and fill out asmts', function() {
             instrumentId: 26363, // Calculation Test
             segmentInterval: 'v3', // can vary this: visit1, v1, v2, v3, v4
             sourceType: '5',  // can vary this: 3, 4, 5, 6, 7, 8, 9
-            assessmentDate: '12/24/1900', // don't change this Date, or need to make sure it's unique in resume queue
-            assessmentStartTime: '10:22',
+            assessmentDate: getRandomDate(),
```

Comment:
Using random date here might cause the failure for the `resume entry` test.  Currently, selenium is using the unique assessment date to grab the assessment to be resumed (I don't know any other way to click the correct link to resume the asmt).  If using random date, when resuming an entry, it might get the wrong asmt.